laurent22/joplin
# All: Fixes #7986: Fix OneDrive sync attempting to call method on `null` variable

# Summary
In some cases, `SyncTargetOneDrive.ts` could reference `this.api_` before initialization to non-null. This seems to cause resources to stop syncing if Joplin was previously closed while syncing a resource.

This fixes #7986 and might also address #7837.

# Testing plan
0. Start with a build of Joplin from before this commit.
1. Follow steps provided in #7986.
2. Apply this commit.
3. Verify that resources download without logging an error message.
